static/src/js/components/FailedToLoadCollectionsMessage/FailedToLoadCollectionsMessage.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { FaBell } from 'react-icons/fa'
+
+import Button from '../Button/Button'
+
+import './FailedToLoadCollectionsMessage.scss'
+
+const FailedToLoadCollectionsMessage = ({ onAlertsClick }) => (
```

Comment:
If granules page fails to load or subscriptions page fails to load will this go off? e.g. [this page](https://search.earthdata.nasa.gov/search/granules?p=C1460503190-GES_DISC&pg[0][v]=f&pg[0][gsk]=-start_date&q=C1460503190-GES_DISC&tl=1427700216.115!5!!)